modules/programs/freetube.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isLinux;
+  # FIXME-REDOX(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isRedox;
+  # FIXME-HURD(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isHurd;
+  # FIXME-FREEBSD(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isFreebsd;
+  inherit (pkgs.stdenv.hostPlatform) isCygwin;
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+  cfg = config.programs.freetube;
+
+  # Config paths are on https://docs.freetubeapp.io/usage/data-location/
+  freetubeConfigPath =
+	if isLinux
+	  # DNM(Krey): Verify that $HOME works
+	  then "$HOME/.config/FreeTube"
+	else if isCygwin
+	  # DNM(Krey): Verify that `%APPDATA%` works
+	  then "%APPDATA%/FreeTube"
+	else if isDarwin
+	  then "$HOME/Library/Application Support/FreeTube/"
+	else
+	  # FIXME-LOCALIZATION(Krey): Integrate `config.i18n.defaultLocale` for translations
+	  throw "Platform '${pkgs.stdenv.hostPlatform}' is not implemented"
+in {
+  meta.maintainers = [ maintainers.kreyren ];
```

Comment:
You need to add yourself to the maintainers list of HM if you are not a maintainer in Nixpkgs' `maintainer-list.nix`.

This is documented in `doc/contributing.adoc`.

``` console
$ nix eval --impure --expr 'with import <nixpkgs> {}; lib.maintainers ? kreyren'
false
```